### PR TITLE
Shirley fixing flashcard next card bug

### PR DESCRIPTION
**Pull request by shirleymango**
- Flashcard Issue: after clicking arrow button to move on to new card, the text does not change, new card is only displayed after user flips the card

https://user-images.githubusercontent.com/44847817/178798782-dfbc5756-7e1a-4219-be79-2a5376fa2007.mov

- Solution: at first I was redrawing new layers with every new card, instead I am now using the same card and changing the text on the layers

- Flashcard Issue 2: after clicking arrow button to move on to new card, the new card is shown but the backside of the new card

https://user-images.githubusercontent.com/44847817/178798844-37cb98d6-f77d-45fc-91dc-1c5681ee4116.mov

- Solution: when loading a new card, check if the current card is flipped; if yes, then flip the card before displaying the new card text

https://user-images.githubusercontent.com/44847817/178798915-76d2ddfb-e146-4a4e-9ce6-f679f95cb412.mov

### CODE REVIEW

File: metau-capstone/StudyViewController.m [hunk 1]
```diff
@@ -108,41 +119,14 @@ - (void) loadFlashcard {
...
-        [label setFontSize:20];
-        [label setString:@"You finished studying today's cards!"];
-        [label setAlignmentMode:kCAAlignmentCenter];
-        label.wrapped = YES;
-        [label setForegroundColor:[[UIColor blackColor] CGColor]];
-        [label setFrame:CGRectMake(0, 0, 300, 180)];
-        [self.front addSublayer:label];
-        
+        [self.frontText setString:@"You finished studying today's cards!"];
         [self.view.layer addSublayer:self.front];
```
Comment (ebeweber-fb) [inline metau-capstone/StudyViewController.m:130]: Way to de-dupe a lot of code here!

File: metau-capstone/StudyViewController.m [hunk 2]
```diff
@@ -31,9 +33,35 @@ @implementation StudyViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     PFUser *user = [PFUser currentUser];
-    // Construct Query
     
-    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"(userID = %@) AND ((levelNum = 1) OR (levelNum = 5))", user.objectId];
+    // Instantiate flashcard sides
```
Comment (ebeweber-fb) [inline metau-capstone/StudyViewController.m:37]: Kudos for pulling this out!

This is probably a good example of something you could put in an instantiateCards method to not entirely bloat the view did load and break out the operations.

**Review by ebeweber-fb — COMMENTED** [review]
Love your diff description!

You can take advantage of markdown + headers to better format your test plan as well. There's a bunch of examples here that you might like :) 

https://axolo.co/blog/p/part-3-github-pull-request-template

A cool thing Github allows you to do is add a template file to the repository (https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository). Then your pull request will be pre-filled with a template you can fill out to keep things consistent.

**Comment by shirleymango** [comment]
The pull request template info is so helpful. Thanks for sharing!

**Pull request merged**